src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,19 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Todos = {
+  todos: Todo[];
+};
+export const TodoList = ({ todos } : Todos) => (
+  <ul>
+    {todos.map(todo => (
+      <li
+        key={todo.id}
+        className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
```

Comment:
you can use classNames function to add dependent class name 
```suggestion
        className={classNames('TodoInfo', { 'TodoInfo--completed', todo.completed })}
```